packages/perseus/src/components/graph.tsx

```diff
@@ -416,7 +416,7 @@ class Graph extends React.Component<Props> {
 
         return (
             <div
-                className="graphie-container above-scratchpad"
+                className="graphie-container blank-background"
```

Comment:
Swapped because `above-scratchpad` adds `blank-background` styles in [perseus-renderer.less](https://github.com/Khan/perseus/pull/2170/files#diff-ff79a578a7a0678d666d4b522615c60cf052a99f70b64109568dd476d9f8e4e4L49)